### ISSUE TITLE
EmptyKtFile should fail if the file is simply a package statement

### ISSUE THREAD

**Issue by efenderbosch-atg** [detekt/detekt#8454]
## Expected Behavior of the rule
```kotlin
package com.example
```
A kt file with ^ that content passes the rule. It should fail.

## Context
Sometimes we'll refactor a couple small, related classes in to a single kt file, like a simple request/response pair. Often the IDE leaves behind a file that only contains the `package` statement, but is otherwise empty. Someone eventually notices and they get deleted, but `EmptyKtFile` flagging this scenario would catch it much earlier.

**Comment by marschwar** [comment]
The `EmptyKotlinFile` rule does just that. There even is a [test](https://github.com/detekt/detekt/blob/main/detekt-rules-empty/src/test/kotlin/dev/detekt/rules/empty/EmptyKotlinFileSpec.kt#L18-L25) that ensures the behavior you are looking for.

Does this not work for you?

**Comment by efenderbosch-atg** [comment]
Slightly confusing that you mention `EmptyKotlinFile` but the docs say `EmptyKtFile`.

But no, it isn't working. Version 1.23.6.

edit: does line 32 need to be `text = text.removeRange(range.startOffset, range.endOffset + 1)`?

KDoc for `CharSequence::removeRange` says
>[endIndex] is not included in the removed part

**Comment by atulgpt** [comment]
Hi @efenderbosch-atg do you want to raise a PR to fix the issue along with the TC?

**Comment by marschwar** [comment]
> Slightly confusing that you mention EmptyKotlinFile but the docs say EmptyKtFile.\n\nBut no, it isn't working. Version 1.23.6.

Sorry for causing confusion. The rule will be renamed in the upcoming 2.x release.

**Comment by efenderbosch-atg** [comment]
> Hi [@efenderbosch-atg](https://github.com/efenderbosch-atg) do you want to raise a PR to fix the issue along with the TC?

I've forked, but build is failing. Any docs on build that I'm not seeing in the README?

Looks like it is trying to fetch libs from maven:
```
Resource missing. [HTTP GET: https://repo.maven.apache.org/maven2/dev/detekt/detekt-cli/1.23.8/detekt-cli-1.23.8.pom]
```

**Comment by atulgpt** [comment]
@efenderbosch-atg they have changed the publishing repo, so might be related to this https://github.com/detekt/detekt/issues/8484

**Comment by marschwar** [comment]
@efenderbosch-atg Please run `./gradlew publishToMavenLocal` 

This is necessary until the artifacts are published with the new coordinates.

**Comment by efenderbosch-atg** [comment]
Green build now.

Added a file w/ just a package declaration and ran Detekt. It caught it. Might have been fixed between `1.23.6` and current `main`? We should be able to upgrade to Kotiln 2.x soon, so we'll bump up to most recent Detekt at that time as well.

**Comment by marschwar** [comment]
You are right. See #6790 

Probably due to the changed name of the rule, the fix was never backported to 1.x.

**Comment by atulgpt** [comment]
> edit: does line 32 need to be `text = text.removeRange(range.startOffset, range.endOffset + 1)`

If the above still an issue you can create a TC and fix the above
